src/common/ant-versions.ts

```
@@ -15,7 +15,12 @@
  */
 import { ANT_REGISTRY_ID } from '../constants.js';
 import { AoANTVersionsRead, AoANTVersionsWrite } from '../types/ant.js';
-import { WithSigner } from '../types/common.js';
+import {
+  AoMessageResult,
+  AoWriteAction,
```

ESLint

[error] 'AoWriteAction' is defined but never used.

(@typescript-eslint/no-unused-vars)